api/src/main/java/eu/coatrack/api/Proxy.java

```diff
@@ -178,4 +178,8 @@ public String toString() {
                 + ", port='" + port + '\''
                 + '}';
     }
+
+    public String getConfigServerName() {
```

Comment:
There is already a getter, called `getCredentialName`. Although that one has an odd name, I would suggest to use that one rather than adding another getter for the same field.

Maybe we should rename that existing getter to "getConfigServerUsername"

Reply:
"Maybe we should rename that existing getter to "getConfigServerUsername""
I agree with that solution. This would eliminate the confusion once and for all.

Reply:
:+1:

Reply:
Done.